docs/_source/conf.py

```diff
@@ -164,9 +164,15 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'Sceptre', 'Sceptre Documentation',
-     author, 'Sceptre', 'One line description of project.',
-     'Miscellaneous'),
+    (
+        master_doc,
+        "Sceptre",
+        "Sceptre Documentation",
+        author,
+        "Sceptre",
+        "One line description of project.",
```

Comment:
Although this line was not introduced here may as well fix it. "Automate cloud deployments" is the tag line on the home page so lets go with that.